src/views/Lottery/components/PastDrawsHistory/PastDrawsHistoryCard.tsx

```diff
@@ -1,18 +1,74 @@
-import React from 'react'
-import { Heading, Card, CardBody } from '@pancakeswap-libs/uikit'
+import React, { useState } from 'react'
+import { Heading, Card, CardBody, Text } from '@pancakeswap-libs/uikit'
 import useI18n from 'hooks/useI18n'
+import styled from 'styled-components'
+import Select, { OptionProps } from 'components/Select/Select'
 import HistoryChart from './HistoryChart'
 import Legend from './Legend'
 
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  flex-direction: column-reverse;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    flex-wrap: nowrap;
+    flex-direction: row;
+  }
+`
+
+const LabelWrapper = styled.div`
+  > ${Text} {
+    font-size: 12px;
+  }
+
+  margin-top: 24px;
```

Comment:
Small thing but can you put component style declarations above children declarations (margin-top as the first entry).

Reply:
done